pkg/config/config.go

```diff
@@ -36,6 +36,16 @@ func LoadConf(bytes []byte) (*sriovtypes.NetConf, error) {
 		return nil, fmt.Errorf("LoadConf(): VF pci addr is required")
 	}
 
+	allocator := utils.NewPCIAllocator(DefaultCNIDir)
```

Comment:
it would be great if you can add UT in config_test.go to check for failure in case device is allocated.

there are tests there already for loadConf